via/CMakeLists.txt

```
@@ -60,9 +59,18 @@ add_executable(vkvia
                   via_system_bsd.cpp)
 endif()
 
+# https://github.com/LunarG/VulkanTools/issues/1908
+find_package(PkgConfig)
+if (PKG_CONFIG_FOUND)
+    pkg_check_modules(jsoncpp IMPORTED_TARGET jsoncpp)
```

```suggestion
    pkg_check_modules(jsoncpp REQUIRED IMPORTED_TARGET jsoncpp)
```
For clearer error in case the pkgconfig for jsoncpp can't be found.